.github/workflows/docs-ci.yaml

```diff
@@ -65,7 +65,8 @@ jobs:
       run: |
         git config user.name nvidia-ci-cd
         git config user.email svc-cloud-orch-gh@nvidia.com
-        gh repo fork --remote --default-branch-only
```

Comment:
No need to fork docs repo, since there is already an existing fork for nvidia-ci-cd user